client/thin-replica-client/include/client/thin-replica-client/grpc_connection.hpp

```diff
@@ -30,6 +31,8 @@ using namespace std::chrono_literals;
 
 namespace client::thin_replica_client {
 
+using RequestId = uint64_t;
```

Comment:
This has nothing to do with the TRC and hence should be defined outside now. Similar to the GrpcConnection object, I suggest to put it under the `client::concordclient` namespace.

Reply:
thanks @robem, I will change this in next commit.